selenium_ex1.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+from selenium import webdriver
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.by import By
+
+
+class SearchText(unittest.TestCase):
+    def setUp(self):
+        self.driver = webdriver.Chrome()                               # create a new Chrome session
+        self.driver.maximize_window()
+        self.driver.get("http://www.google.com/")                    # navigate to the google home page
+
+
+    def test_search_automation(self):
+        self.assertTrue(self.is_element_present(By.NAME, 'q'))   # check search box exists on Home page
+        self.assertTrue(self.is_element_present(By.NAME,'btnI'))    #Name of the I'm Feeling Lucky box
+        self.assertTrue(self.is_element_present(By.CLASS_NAME, 'hb2Smf'))    #Check's the class name for "search by mic"
+        self.assertTrue(self.is_element_present(By.CLASS_NAME,'gNO89b'))        #Check's Google Search-box class name
+        self.assertTrue(self.is_element_present(By.CLASS_NAME, 'gb_Of'))            # Check's for the google apps square dots near google account
+        self.search_field = self.driver.find_element_by_name('q')    # get the search textbox
+        self.search_field.send_keys('Selenium WebDriver')            # enter search keyword and submit
+        self.search_field.submit()
+        self.driver.find_element_by_class_name('ellip').click()      #First result on google search (the class name of Selenium webdriver)
```

Comment:
All classes in the urls list have are the same. if by relevance, the link you want to click is the second in the list, you will click on another link. you need to click on the exact link.